documentation/5_Organizational_Agreements.md

```diff
@@ -65,4 +65,162 @@ process is defined by the IDSA certification working group.
+under an overarching integrated governance approach:
+
+- *Technical level,* to provide software and hardware components for
+    controlled, sovereign and secure sharing of data.
+
+- *Semantic level,* to ensure that format and meaning of shared data
+    is preserved and understood.
+
+- *Organizational level,* to let stakeholders align goals,
+    expectations, responsibilities, and business processes.
+
+- *Legal level,* to make sure that organizations under different legal
+    jurisdictions and frameworks can share data with common legally
+    binding conditions.
+
+Governance on the identified topics distinguishes between two
+development lines for data space instances:
+
+- *Intra data space interoperability,* between the data space
+    authority, processing, and data sharing building blocks within a
+    single data space instance.
+
+- *Inter data space interoperability,* between multiple data space
+    instances at each of the functional levels as distinguished in the
+    framework shown in Figure 2.
+
+To enable interoperability between data spaces, each of the functional
+levels shown contains topics that require adequate
+governance. For each of the levels, these topics are identified in the
+following paragraphs. Their governance aspects are addressed in the next
+chapter.
+
+In general, the participant in a data space covers the technical and
+semantic interoperability aspects, while the Data Space Governance
+Authority defines the organizational and legal aspects.
+Based on the governance scheme, the participants participate in the
+defintion of the policies and rules of a data space, including the
+framework or agreements on all four levels.
+
+### Technical level
+
+The technical level covers the software and hardware components for
+controlled, sovereign and secure sharing of data. It consists of five
+sublevels with the topics that require adequate governance:
+
+### Governance instruments
+
+The various governance instruments that may be considered by the IDSA
+are listed in the Table below. These governance tools each describe a
+different aspect of a certain activity and are used in the next section
+to provide a multidimensional overview of the intra and inter data
+spaces governance.
+
+| **Governance Instrument** | **Governance instrument description**      |
+|--------------------------------- |---------------------------------|
+| *Standardization* | Ensuring that the tasks, processes, and guidelines around this activity are formalized, documented, and  aligned between data spaces instances and the IDSA. The standardization efforts can generally be used as a  blueprint or starting point for the stakeholders.|
+| *Certification* | Validating that stakeholders act according to the standardized way of working. Certification is divided into component certification (certification of technical/software components) and organizational certification (organizational and legal processes).    |
```

Comment:
It's not Certification, but rather Compliance (with standardized protocols). IDSA will ensure that Certification includes compliance with the protocols, but Certification covers a lot more than interoperability on a technical level.